trustpinning/certs.go

```diff
@@ -112,7 +112,7 @@ func ValidateRoot(prevRoot *data.SignedRoot, root *data.Signed, gun data.GUN, tr
 	havePrevRoot := prevRoot != nil
 	if havePrevRoot {
 		// Retrieve all the trusted certificates from our previous root
-		// Note that we do not validate expiries here since our originally trusted root might have expired certs
+		// Note that we do not validate expires here since our originally trusted root might have expired certs
```

Comment:
expiries is correct.

Reply:
Thank you very much, this has been reverted.